src/sentry/newsletter/base.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+
+class Newsletter(object):
+    PRODUCT_CAMPAIGN = 1
```

Comment:
@mattrobenolt in analytics and advertising terms campaign is infinite. They're generated whenever and wherever they're needed. Not that this would happen, but maybe we decided we wanted a way for someone else's blog to allow people to subscribe. 

If campaign literally means "what are they subscribed to" thats different, but I wouldn't think of it as a campaign at that point, but rather a multiple choice list of subscriptions.